tests/serverpod_test_server/test/generated_methods/copy_with_mutable_test.dart

```diff
@@ -235,5 +235,110 @@ void main() {
       expect(mapDataCopy.data['b']?.num, 2);
       expect(mapDataCopy.data['b']?.id, null);
     });
+
+    group('Given an object with nested Lists and Maps when calling copyWith',
+        () {
+      var objectWithNestedObjects = ObjectWithObject(
+        data: SimpleData(num: 1),
+        dataList: [SimpleData(num: 2)],
+        listWithNullableData: [],
+        nestedDataListInMap: {
+          'firstKey': [
+            [
+              {111: SimpleData(num: 111)}
+            ],
+            [
+              {222: SimpleData(num: 222)}
+            ],
+          ],
+        },
+        nestedDataList: [
+          [SimpleData(num: 88), SimpleData(num: 99)],
+        ],
+        nestedDataMap: {
+          'firstKey': {
+            333: SimpleData(num: 333),
+          },
+        },
+      );
+
+      test('then a nested map should be deeply cloned', () {
```

Comment:
Suggestion: Change description to `then nested map IS deeply cloned`.

I think it is more clear to state the expect rather than the intent.